source/dataman/export/AMExporterOptionGeneralAscii.cpp

```diff
@@ -47,6 +49,75 @@ AMExporterOption *AMExporterOptionGeneralAscii::createCopy() const
 	option->dissociateFromDb(true);
 	return option;
 }
+
+int AMExporterOptionGeneralAscii::exportPrecision() const
+{
+	return exportPrecision_;
+}
+
+int AMExporterOptionGeneralAscii::exportPrecision(const QString &source) const
+{
+	if(sourceExportPrecision_.contains(source)){
+		return sourceExportPrecision_.value(source);
+	}
+	else
+		return exportPrecision_;
+}
+
+void AMExporterOptionGeneralAscii::setExportPrecision(const QString &source, int precision)
+{
+	if(!source.isEmpty() && precision >= 1) {
+		sourceExportPrecision_.insert(source, precision);
+	}
+}
+
+void AMExporterOptionGeneralAscii::setExportPrecision(int precision)
+{
+	if(precision >= 1){
+		exportPrecision_ = precision;
+		setModified(true);
+	}
+}
+
+QString AMExporterOptionGeneralAscii::loadPrecisionMap()
+{
+	QStringList pm;
```

Comment:
I know it's just an internal variable that is not too hard to figure out, but it's still always preferable to use verbose names.